spec/javascripts/health-care/health-care.spec.js

```diff
@@ -192,6 +192,15 @@ describe("Health Care Form", function() {
     form.elements['veteran[city_of_birth]'].value = "Kent";
     form.elements['veteran[state_of_birth]'].value = "WA";
     form.elements['veteran[email]'].value = "test@foo.com";
+    form.elements['veteran[gross_wage_income]'].value = "55000.00";
+    form.elements['veteran[net_business_income]'].value = "1200.00";
+    form.elements['veteran[other_income]'].value = "123.00";
+    form.elements['veteran[spouses][gross_wage_income]'].value = "60000.00";
+    form.elements['veteran[spouses][net_business_income]'].value = "2400.00";
+    form.elements['veteran[spouses][other_income]'].value = "234.00";
+    form.elements['veteran[children][gross_wage_income]'].value = "54.00";
+    form.elements['veteran[children][net_business_income]'].value = "43.00";
+    form.elements['veteran[children][other_income]'].value = "32.00";
```

Comment:
Since you added new validation functions, and since those functions contain non-trivial logic, we should have 1 unittest per function.  That also means we'll need to expose the the functions for testing, but shouldn't be too hard.